product_extended_segmentation/model/company.py

```diff
@@ -7,6 +7,6 @@ class ResCompany(models.Model):
     _inherit = 'res.company'
 
     std_price_neg_threshold = fields.Float(
-            string="Standard Price Negative Threshold (%)",
-            help=('Maximum percentage threshold that Standard Price is '
-                  'allowed to lower'))
+        string="Standard Price Bottom Threshold (%)",
+        help=('Maximum percentage threshold that Standard Price is '
+              'allowed to lower'), default=1.0)
```

Comment:
'Maximum percentage threshold that Standard Price is '
'allowed to be lower in order to be changed with the update cost wizard'